src/app/auth/auth-error/auth-error.component.ts

```diff
@@ -7,6 +7,8 @@ import {
 import { HttpParams } from '@angular/common/http';
 import { HTTP_PARAMS } from '../http-params';
 
+const BASE_URL = 'https://docs.tf2pickup.org/player-registration-issues';
```

Comment:
Don't use uppercase; use camelCase for variables and consts.
Also, make this variable name more readable, e.g. `docsBaseUrl`.

Reply:
changed